src/WebJobs.Script.WebHost/Web.config

```diff
@@ -181,7 +181,7 @@
       <dependentAssembly>
         <assemblyIdentity name="System.IO.Compression" publicKeyToken="b77a5c561934e089" culture="neutral" />
         <!--Explicitly redirecting back to 4.0.0. See https://github.com/dotnet/corefx/issues/15693 for details-->
-        <bindingRedirect oldVersion="0.0.0.0-4.1.2.0" newVersion="4.0.0.0" />

```

Comment:
d'oh. gets me every time. Is there some test we can add for this? Do an xpath on the XML file?